src/components/titledLists/TitledList.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Typography, List, ListItem, ListItemText, Box } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+function TitledList({ title, list }) {
+  const theme = useTheme();
+
+  return (
+    <Box>
+      <Typography
+        variant="h6"
+        gutterBottom
+        sx={{ color: theme.palette.bright.main }}
+      >
+        {title}
+      </Typography>
+      <List>
+        {list.map((item, index) => (
+          <ListItem key={index} sx={{ padding: "0" }}>
```

Comment:
using "index" as a list key is not preferable. 
would you please make sure to change it, as I believe there is a unique id for each item on the list that you will display. 
However, if the list has static data and you are sure that it wouldn't change, then it's fine to use the index.

Reply:
Check this article:
https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

Reply:
Yes, it is static data, that is why I am using index.